app/controllers/admin_controller.rb

```diff
@@ -63,20 +61,6 @@ def plugins
     @plugins = Redmine::Plugin.all.sort
   end
 
-  # Loads the default configuration
-  # (roles, types, statuses, workflow, enumerations)
-  def default_configuration
-    if request.post?
-      begin
-        Redmine::DefaultData::Loader::load(params[:lang])
-        flash[:notice] = l(:notice_default_data_loaded)
-      rescue => e
-        flash[:error] = l(:error_can_t_load_default_data, ERB::Util.h(e.message))
```

Comment:
I think you did forget to remove the localization strings `error_can_t_load_default_data` and `notice_default_data_loaded` from the `en.yml`.